src/Microsoft.AspNetCore.Mvc.Razor/Internal/CSharpCompiler.cs

```diff
@@ -15,7 +15,9 @@ public class CSharpCompiler
         private readonly CSharpCompilationOptions _compilationOptions;
         private readonly CSharpParseOptions _parseOptions;
         private readonly RazorReferenceManager _referenceManager;
-        private readonly DebugInformationFormat _pdbFormat = DebugInformationFormat.PortablePdb;
+        private readonly DebugInformationFormat _pdbFormat = SymbolsUtility.SupportsFullPdbGeneration() ?
+            DebugInformationFormat.Pdb :
```

Comment:
This got removed as part of the netcoreapp2.0 conversion (https://github.com/aspnet/Mvc/commit/1c5e4176069ea20671e1738ac599e544633f47ce#diff-584adecdd95f7d31bf6950c51109cb36L17). I assumed bringing back portable pdb reader would suffice, but that solves a different problem.